plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml

```diff
@@ -294,4 +294,16 @@
     <extension point="org.eclipse.help.contexts">
        <contexts file="helpContexts.xml" />
     </extension>
+	<extension point="org.eclipse.ui.perspectiveExtensions">
```

Comment:
Indent looks off. Are you using tabs? We try to stick to spaces.

Reply:
Yes,  done format with default settings. Updated to two spaces